src/routes/login/components/Login.js

```diff
@@ -93,45 +95,78 @@ module.exports = class Login extends StrangeForms(React.Component) {
 
         return (
 
-            <form onSubmit={this.submit}>
-                <h2>Login</h2>
-                <div>
-                    <label>Email:</label>
-                    <input
-                        id='email'
-                        value={this.fieldValue('email')}
-                        onChange={this.proposeNew('email')}
-                        onBlur={this.fieldBlurred}
-                    />
-                    {this.showEmailError() && <label style={{ color:'red' }}>Please enter a valid email address</label>}
-                </div>
-                <div>
-                    <label>Password:</label>
-                    <input
-                        type='password'
-                        value={this.fieldValue('password')}
-                        onChange={this.proposeNew('password')}
-                    />
-                </div>
-                <label>
-                    <input
-                        type='checkbox'
-                        checked={this.fieldValue('rememberMe')}
-                        onChange={this.proposeNew('rememberMe')}
-                    />
-                    Remember Me
-                </label>
-                <p>Don&rsquo;t have an account? <NavLink to='sign-up'>Sign up</NavLink> now.</p>
-                <p><NavLink to='forgot-password'>Forget password?</NavLink></p>
-                {this.props.errorMessage &&
-                    <div style={{ color: 'red' }}>Error! {this.props.errorMessage}</div>
-                }
-                <button
-                    type='submit'
-                    onClick={this.submit}
-                    disabled={this.disableButton()}
-                >Login</button>
-            </form>
+            <FormWrapper>
+                <form onSubmit={this.submit}>
+                    <TextWrapper>
+                        <Typography variant='headline' gutterBottom>Log In</Typography>
+                        {this.props.errorMessage &&
+                            <FormHelperText>Oops, something went wrong! {this.props.errorMessage}</FormHelperText>
```

Comment:
It would be helpful to have a language style guide for messages and titles. Elsewhere this reads "Error: *message*," and it should be consistent throughout the site. I'm not a big fan of, "Oops," but I'm also not much fun in general.

Reply:
Yeah it'd be cool to have a constants file that has error prefixes depending on which type of error it is

Reply:
Yep that's my fault! The message formatting depends on my mood for the day....I changed this to "oops" while adding material cause it felt more "fun" and human, less developer-y; obviously didn't carry that over to the other components

Reply:
Humans are fun 🤖⚡️👽